SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -99,17 +113,38 @@ public final class SwiftGenStoryboardEnumBuilder {
             
             text += "\(s)\(t)}\n"
         }
-        if enumName != nil {
-            text += "\(t)}\n"
+        text += "\(t)}\n"
+      
+        /// Segues
+        if (storyboardsSegues.count > 0) {
+          text += "\n"
+          
+          text += "\(t)enum \(seguesEnumName.asSwiftIdentifier()) {\n"
+        
+          for (name, segues) in storyboardsSegues
+            where segues.count > 0 {
+              
+            let enumName = name.asSwiftIdentifier(forbiddenChars: "_")
+            text += "\(s)\(t)enum \(enumName) : String {"
+            
+            for segue in segues {
+              let caseName = segue.segueID.asSwiftIdentifier(forbiddenChars: "_")
+              text += "\n\(s)\(t)\(t)/// \(caseName)\n"
+              text += "\(s)\(t)\(t)case \(caseName)Segue = \"\(caseName)\"\n"
```

Comment:
**Don't** take out the assignment but rather fix it instead: missed that earlier but in fact it should `= "\(segue.segueID)"` instead (in case the `asSwiftIdentifier(…)` method did remove some special chars invalid for an identifier but you still need to associate the real identifier string to that enum)